data/nav_graphql.yml

```diff
@@ -428,12 +430,6 @@
           path: apis/graphql/schemas/object/organizationmembersso
         - name: OrganizationMemberSecurity
           path: apis/graphql/schemas/object/organizationmembersecurity
-        - name: OrganizationMemberSuite
-          path: apis/graphql/schemas/object/organizationmembersuite
-        - name: OrganizationMemberSuiteConnection
-          path: apis/graphql/schemas/object/organizationmembersuiteconnection
-        - name: OrganizationMemberSuiteEdge
-          path: apis/graphql/schemas/object/organizationmembersuiteedge
```

Comment:
Are these breaking changes for users? Has it already been communicated in a changelog entry?

Reply:
That's a great point. I had a skim over the changelog and didn't see anything about it. It would definitely be worth communicating these kind of changes in the future.